### ISSUE TITLE
`logits` and `labels` must have the same shape, received ((32, 5) vs (32, 1)).

### ISSUE THREAD

**Issue by DS-io** [fchollet/deep-learning-with-python-notebooks#216]
PLS HELP ME TO SORT THIS THING OUT
![1](https://user-images.githubusercontent.com/93241103/212591555-e74f54e8-0849-4334-a9d8-6098a9f81b71.png)
![2](https://user-images.githubusercontent.com/93241103/212591565-2d35f7a0-acd9-4a3a-8fca-185a3d1f7ab5.png)
![3](https://user-images.githubusercontent.com/93241103/212591572-293e5254-29af-4801-97b6-d00f764e2e07.png)
